rdbms/src/main/java/org/spine3/server/storage/jdbc/query/tables/commands/CreateTableIfDoesNotExistQuery.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2016, TeamDev Ltd. All rights reserved.
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package org.spine3.server.storage.jdbc.query.tables.commands;
+
+import org.spine3.server.storage.jdbc.DatabaseException;
+import org.spine3.server.storage.jdbc.query.AbstractQuery;
+import org.spine3.server.storage.jdbc.query.constants.CommandTable;
+import org.spine3.server.storage.jdbc.util.ConnectionWrapper;
+
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+
+public class CreateTableIfDoesNotExistQuery extends AbstractQuery {
+
+    @SuppressWarnings("DuplicateStringLiteralInspection")
+    private static final String INSERT_QUERY =
+            "CREATE TABLE IF NOT EXISTS " + CommandTable.TABLE_NAME + " (" +
+                    CommandTable.ID_COL + " VARCHAR(512), " +
+                    CommandTable.COMMAND_COL + " BLOB, " +
+                    CommandTable.COMMAND_STATUS_COL + " VARCHAR(512), " +
+                    CommandTable.ERROR_COL + " BLOB, " +
+                    CommandTable.FAILURE_COL + " BLOB, " +
+                    " PRIMARY KEY(" + CommandTable.ID_COL + ')' +
+                    ");";
+
+    public CreateTableIfDoesNotExistQuery(Builder builder) {
+        super(builder);
+    }
+
+    public static Builder getBuilder() {
+        final Builder builder = new Builder();
+        builder.setQuery(INSERT_QUERY);
+        return builder;
+    }
+
+    public void execute() throws DatabaseException {
+        try (ConnectionWrapper connection = dataSource.getConnection(true);
+             PreparedStatement statement = this.prepareStatement(connection)) {
+            statement.execute();
+        } catch (SQLException e) {
+            //log().error("Exception during table creation:", e);
```

Comment:
Add TODOs for such commented code, to remember to fix it later.
Make log() package local in the storage.